src/librustc_driver/driver.rs

```diff
@@ -150,40 +140,43 @@ pub fn compile_input(sess: Session,
                                     control.make_glob_map,
                                     |tcx, mir_map, analysis| {
 
-            {
-                let state = CompileState::state_after_analysis(input,
-                                                               &tcx.sess,
-                                                               outdir,
-                                                               &expanded_crate,
-                                                               tcx.map.krate(),
-                                                               &analysis,
-                                                               tcx,
-                                                               &lcx,
-                                                               &id);
-                (control.after_analysis.callback)(state);
-
-                tcx.sess.abort_if_errors();
-                if control.after_analysis.stop == Compilation::Stop {
-                    return Err(());
-                }
-            }
-
-            if log_enabled!(::log::INFO) {
-                println!("Pre-trans");
-                tcx.print_debug_stats();
-            }
-            let trans = phase_4_translate_to_llvm(tcx, &mir_map, analysis);
-
-            if log_enabled!(::log::INFO) {
-                println!("Post-trans");
-                tcx.print_debug_stats();
-            }
-
-            // Discard interned strings as they are no longer required.
-            token::get_ident_interner().clear();
-
-            Ok((outputs, trans))
-        })
+                                        {
```

Comment:
Seems like maybe the issue where large closures should be block indented rather than visual? I couldn't find the issue for it, but I know there is one!